apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -768,19 +762,51 @@ export async function updateLti13Scores(
     }),
   );
 
+  const course_staff_data = await selectUsersWithCourseInstanceAccess({
+    course_instance_id: assessment.course_instance_id,
+    minimal_role: 'Student Data Viewer',
+  });
+  const course_staff = new Set(course_staff_data.map((staff) => staff.uid));
```

Comment:
Two naming nits:

- We generally reserve `snake_case` for things that map directly to database tables/columns with snake-cased names.
- When dealing with DB things, names like `courseStaff` would generally indicate a model or list of models, not a set of strings.

```suggestion
  const courseStaff = await selectUsersWithCourseInstanceAccess({
    course_instance_id: assessment.course_instance_id,
    minimal_role: 'Student Data Viewer',
  });
  const courseStaffUids = new Set(course_staff_data.map((staff) => staff.uid));
```

(plus corresponding downstream renames, of course.

Reply:
Addressed in 725e64af39